### DISCUSSION TITLE
Cannot find the 'Sync Ldap Groups to keycloak' option

### DISCUSSION THREAD

**Discussion by nvp152** [keycloak/keycloak#15220] [Archive]
Hi,

According to older documentation (see the screenshots  [here](https://www.janua.fr/mapping-ldap-group-and-roles-to-redhat-sso-keycloak/) ) there used to be a 'Sync ldap groups to keycloak' off of the ldap group-mapper pane. I am not finding it in version  19.0.3. Anyone?

**Comment by jhollmannk** [comment]
It's hidden under the "Action" Drop-Down (upper right corner) of the group mapper editing page.
